.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+**Task name**: <Name of changed or new azure pipeline task>
+
+**Description**: <Describe your changes here>
+
+**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>
+
+**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>
+
+**Attached related issue:** (Y/N) <Please add link to related issue here. We are not going to close issue until changes will be deployed, so please don't link it directly to PR>
```

Comment:
The linking an issue to a PR vs waiting for deployment is specific to our team so I would leave that sentence out. Other teams might close issues before deployment completes

Reply:
Yep 👍 Thanks, fixed